lib/editor/containers/ActiveEntityList.js

```diff
@@ -53,9 +63,11 @@ const mapStateToProps = (state: AppState, ownProps: Props) => {
 
 const mapDispatchToProps = {
   cloneGtfsEntity,
+  createSnapshot,
   deleteGtfsEntity,
   enterTimetableEditor,
   newGtfsEntity,
+  patchTable,
```

Comment:
Bloated container alert! Could use refactoring of sub-components to not require passing through so many props.